docs/developer/architecture.rst

```diff
@@ -168,23 +168,41 @@ The base storage format in Modin is pandas. In the default case, the Modin Dataf
 Data Ingress
 ''''''''''''
 
+.. note::
+   Data ingress operations (e.g. ``read_csv``) in Modin load data from the source into
+   partitions and vice versa for data egress (e.g. ``to_csv``) operation.
+   Improved performance is achieved by reading/writing in partitions in parallel.
+
+The data ingress operations workflow diagram is shown below. After a data 
+ingress function from pandas API (e.g. ``read_csv``) is called, the compiled
+query from pandas API is passed to the :doc:`Factory Dispatcher </flow/modin/core/execution/dispatching>`,
```

Comment:
Inside flow/modin/core/execution/dispatching.rst there is the following sentence:
```
modin.pandas.read_csv calls FactoryDispatcher.read_csv, which calls .read_csv function of the factory...
```
Please change `.read_csv` to `._read_csv` function of the factory.

Reply:
Fixed.